adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -92,6 +128,32 @@ def connect(self):
                     self.reset()
                 continue
 
+    def connect_enterprise(self):
+        """
+        Attempt an enterprise style WiFi connection
+        """
+        self._esp.wifi_set_network(bytes(self.ent_ssid, 'utf-8'))
+        self._esp.wifi_set_entidentity(bytes(self.ent_ident, 'utf-8'))
+        self._esp.wifi_set_entusername(bytes(self.ent_user, 'utf-8'))
+        self._esp.wifi_set_entpassword(bytes(self.ent_password, 'utf-8'))
+        self._esp.wifi_set_entenable()
+        while not self._esp.is_connected:
+            try:
+                if self.debug:
+                    print("Waiting for the ESP32 to connect to the WPA2 Enterprise AP...")
+                self.pixel_status((100, 0, 0))
+                sleep(1)
+                failure_count = 0
```

Comment:
This is a bit different form the "normal" connect since the actual connect is not in the try/except loop. It's not clear to me how this works...I looks like it just waits for a connection but as constructed, I don't think it will ever time out and it is not clear that the failure_count test is meaningful. 
An exception in lines 135-139 with not be "handled" and will simply raise the exception. Is that the desired response?

Or I am just confused...

Reply:
Fixed in the latest commit.